src/frontends/tests/frontend/shared/src/cut_specific_model.cpp

```diff
@@ -56,7 +56,7 @@ std::vector<ov::frontend::Place::Ptr> FrontEndCutModelTest::constructNewOutputs(
 
 ///////////////////////////////////////////////////////////////////
 
-TEST_P(FrontEndCutModelTest, testOverrideInputs) {
+TEST_P(FrontEndCutModelTest, DISABLED_testOverrideInputs) {
```

Comment:
https://github.com/meiyang-intel/openvino/commit/e619a40682f393c6885cf6cea72b717cc994b052 can fix this issue. Please help to merge.

Reply:
merged, thanks for your help.